pom.xml

```diff
@@ -64,8 +64,7 @@
     <module>standalone-metastore</module>
     <module>upgrade-acid</module>
     <module>kafka-handler</module>
-    <module>iceberg-handler</module>
-    <module>iceberg-catalog</module>
+    <module>iceberg</module>
```

Comment:
Just to check, after this refactor, the handler and catalog jars are still packaged into the `packaging/target/.../lib` directory? How about the patched jars (core and api)? Are they correctly included into the shaded handler jar with the patched classfiles, but the patches jars are not included into the packaging `/lib`?

Reply:
That's a good question!
I will check...

Reply:
Applied my other patches so there is a new class in `patched-iceberg-api` (`CommitStateUnknownException`)
Run the packaging and checked the `packaging/target/.../lib` and it still contains the libs:
```
$ cd packaging/target/apache-hive-4.0.0-SNAPSHOT-bin/apache-hive-4.0.0-SNAPSHOT-bin/
$ ls *iceberg*
hive-iceberg-catalog-4.0.0-SNAPSHOT.jar	hive-iceberg-handler-4.0.0-SNAPSHOT.jar
```

The new class is in the jar:
```
$ zip -sf hive-iceberg-handler-4.0.0-SNAPSHOT.jar |grep CommitStateUnknownException
  org/apache/hive/iceberg/org/apache/iceberg/exceptions/CommitStateUnknownException.class
```

So this seems ok

Reply:
Great, thanks